tpcc/worker.go

```diff
@@ -189,14 +189,14 @@ func (w *worker) run(errCh chan<- error, warehouseID int) {
 			if firstRun {
 				// Sleep for a random duration up to the keying time to smooth out any
 				// potential thundering herd effects when the load generator starts.
-				sleepTime = time.Duration(float64(sleepTime)*rand.Float64()) + t.randThinkTime()/2
+				sleepTime = time.Duration(float64(sleepTime) * rand.Float64())
```

Comment:
Can we ditch this `firstRun` stuff now that there's a ramp up time?

Reply:
I've added a TODO to revisit this. I don't want to rip it out without testing and I won't be able to test until later tonight or tomorrow.